style/style.css

```diff
@@ -6,72 +6,225 @@ Version: 1.0
 Author: Pepijn de Vos, Pepijn Goorden
 Author URI: http://pepijndevos.nl/
 */
+:root{
+    --background: #FFF;
+    --text: #151515;
+    --grey: lightgray;
+    --theme: #227;
+    --theme-light: rgba(240, 240, 247, 1); /* #22227711; */
+    --main-width: 620px;
+    --padding: 3em;
+    --serif: Newsreader;
+    --sans: Hind;
```

Comment:
So what do I put here to reproduce the different options you were playing with?

Reply:
the --serif will change the body text, the --sans will change the sidebar and data/tag fonts